core/validatorapi/validatorapi.go

```diff
@@ -357,6 +360,66 @@ func (c Component) SubmitBeaconBlock(ctx context.Context, block *spec.VersionedS
 	return nil
 }
 
+// SubmitVoluntaryExit receives the partially signed voluntary exit.
+func (c Component) SubmitVoluntaryExit(ctx context.Context, ve *eth2p0.SignedVoluntaryExit) error {
+	vs, err := c.Validators(ctx, "head", []eth2p0.ValidatorIndex{ve.Message.ValidatorIndex})
+	if err != nil {
+		return err
+	}
+	if len(vs) != 1 {
```

Comment:
can do `else if` since it is part of the error handling/validator of the above call.

Reply:
I have changed this code, so this `if len' is not anymore